config/checkstyle/checkstyle.xml

```diff
@@ -124,7 +124,7 @@
         <!-- Checks for Size Violations.                    -->
         <!-- See http://checkstyle.sf.net/config_sizes.html -->
         <module name="LineLength">
-            <property name="max" value="120"/>
```

Comment:
I agree, 120 is way too low for current displays and the need for wrapping single line to multiple lines in most cases leads to worse code readability.